### ISSUE TITLE
Multiline calc() are not calculated

### ISSUE THREAD

**Issue by alienlebarge** [vdch/regenair-fabricator#156]
## Description

We use [postcss-calc] to translate `calc()` to proper value that browser that doesn't support it will understand.

But it seems that when `calc()` is written on more than one line, _postcss-calc_ doesn't work.

See postcss/postcss-calc#26 for more infos
## Ask

Translate `calc()` to understandable content for old browsers.
## To Do
- [x] Open a issue on postcss-calc repo
- [x] Use a workaround in the meantime of a fix
## Additional Info

[`calc()` support on CanIUse.com](http://caniuse.com/#search=calc%28%29)
